### DISCUSSION TITLE
What is the rule of the milvus collection/partition name?

### DISCUSSION THREAD

**Discussion by shanghaikid** [milvus-io/milvus#6171] [Q&A and General discussion]
How many characters? Which kind special chars support? length? I can not find it in the milvus.io.

**Comment by yhmo** [comment]
Rule for collection/partition name:
- Collection name cannot be empty.
- Collection name length <= 255.
- Collection name first character must be underscore or character(a ~ z, A ~ Z).
- Collection name can only contain numbers, letters, and underscores(_, 0 ~9, a ~ z, A ~Z).
